django/cantusdb_project/main_app/views/chant.py

```diff
@@ -358,12 +358,45 @@ def get_context_data(self, **kwargs):
             # https://cantusindex.org/json-con/{cantus_id}/refresh in a new thread,
             # thus ensuring the current page doesn't take a long time to load, while also
             # ensuring that concordances are up-to-date for future page loads.
+
+            def refresh_ci_json_con_api(cantus_id: str) -> None:
+                """
+                Send a request to CantusIndex's "refresh" json-con API, causing it to
+                update its cached concordances for the specified Cantus ID. This
+                function is meant to be run in a separate thread, so all warnings/errors
+                are suppressed.
+
+                Args:
+                    cantus_id (string): a Cantus ID
+
+                Returns:
+                    None
+                """
+                try:
+                    requests.get(
+                        url=f"https://cantusindex.org/json-con/{chant.cantus_id}/refresh",
+                        timeout=1,
+                        verify=False,  # since we're not doing anything with the data from
```

Comment:
I get why we don't *need* to have a valid SSL, but what's the advantage of not doing the verification?

Reply:
I've removed this argument - good suggestion!